lib/sdk.ts

```diff
@@ -108,6 +108,10 @@ class OptableSDK {
     return email ? "e:" + sha256.hex(email.toLowerCase().trim()) : "";
   }
 
+  static sha256(string: string): string {
+    return string ? sha256.hex(string) : "";
```

Comment:
should we try to eliminate the usual bad values such as "null" "0" "false" "undefined" ?